packages/rum-core/src/domain/view/viewMetrics/trackCumulativeLayoutShift.spec.ts

```diff
@@ -21,13 +16,7 @@ describe('trackCumulativeLayoutShift', () => {
 
   function newLayoutShift(lifeCycle: LifeCycle, overrides: Partial<RumLayoutShiftTiming>) {
     lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED, [
-      {
-        entryType: 'layout-shift',
-        startTime: relativeNow(),
-        hadRecentInput: false,
-        value: 0.1,
-        ...overrides,
-      },
+      createPerformanceEntry(RumPerformanceEntryType.LAYOUT_SHIFT, overrides),
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏what about removing the `newLayoutShift` helper? it does not seem to bring a lot now